hack/run-e2e.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+for i in test/e2e/*.bats; do
+	./test/e2e/bats/core/bin/bats ${i}
```

Comment:
nit: Here we could use `/test/e2e/bats/core/bin/bats --recursive test/e2e/*.bats` instead.